data/org.freedesktop.portal.Notification.xml

```diff
@@ -66,21 +68,69 @@
 
           User-visible string to display as the body.
 
+        * ``markup-body`` (``s``)
+
+          The same as ``body`` but with support for markup formatting.
+          The markup is XML-based and supports a small subset of HTML
+          ``<b>...</b>``, ``<i>...</i>`` and ``<a href="...">...</a>``.
```

Comment:
I excluded <u>  on purpose since it's non what most people think since the definiton changed in html5 https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u

Adding multiple inline images leeds probably to more problems then it solves. If you look at the full specs https://github.com/flatpak/xdg-desktop-portal/pull/1304 we will have one body image.
We can extend the list later or even maybe add CSS styling.

Adding

Reply:
> I excluded on purpose since it's non what most people think since the definiton changed in html5 https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u

I'm doubt anyone is going to get full HTML support in notifications and with simple parsers it has sense to do custom tags for markup.

HTML itself has web components thing that allow one to create custom tags.

Reply:
Your points are valid. We can add this for a later version.